COPYRIGHT

```diff
@@ -1,4 +1,21 @@
-© 2010 Battelle Energy Alliance, LLC
+Large portions of this software were developed at the Idaho National Laboratory
+
+Major contributors include the following:
+The Commonwealth Scientific and Industrial Research Organisation
+
+Other contributors include the following:
+The University of Texas at Austin
+Argonne National Laboratory
+Los Alamos National Laboratory
+Oak Ridge National Laboratory
+Massachusetts Institute of Technology
+
+For complete author information reference the Git repository
+https://github.com/idaholab/moose
+
+The following are the complete copyright notices for select institutions
+
+© 2010-2018 Battelle Energy Alliance, LLC
```

Comment:
Looks like this copyright symbol has always been a Unicode character. Not sure if we care about this/want to change it?

Reply:
Yeah, I'm not sure I really care about it either way unless you feel strongly about it.